janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/ElasticSearchConfigTest.java

```diff
@@ -50,20 +67,37 @@
 
     private static final String INDEX_NAME = "escfg";
 
+    private static final String ANALYZER_KEYWORD = "keyword";
+
+    private static final String ANALYZER_ENGLISH = "english";
+
+    private static final String ANALYZER_STANDARD = "standard";
+
     private ElasticsearchRunner esr;
 
     private int port;
 
+    private HttpHost host;
+
+    private CloseableHttpClient httpClient;
+
     @Before
     public void setup() throws Exception {
         esr = new ElasticsearchRunner();
         esr.start();
         port = getInterface() == ElasticSearchSetup.REST_CLIENT ? 9200 : 9300;
+        httpClient = HttpClients.createDefault();
+        try {
+            host = new HttpHost(InetAddress.getByName("127.0.0.1"), 9200);
```

Comment:
Yes it's ok. Even if JanusGraph talks to Elasticsearch with the tranport API over port 9300, the mapping or the template are pushed with the REST API via 9200 port.